pkg/pod/ttlDestroyHandler_test.go

```diff
@@ -0,0 +1,59 @@
+package pod
+
+import (
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/dhis2-sre/rabbitmq/pgk/queue"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type mockQueueProducer struct{ mock.Mock }
+
+func (m *mockQueueProducer) Produce(channel queue.Channel, payload any) {
+	m.Called(channel, payload)
+}
+
+func Test_ttlDestroyHandler_Handle(t *testing.T) {
+	producer := &mockQueueProducer{}
+	handler := NewTTLDestroyHandler(producer)
+	pod := v1.Pod{
+		ObjectMeta: metav1.ObjectMeta{
+			Labels: map[string]string{
+				"im-id":                 "1",
+				"im-creation-timestamp": strconv.Itoa(int(time.Now().Add(time.Minute * 5).Unix())),
```

Comment:
This expression is a bit hard to read 😅 

Since the test is just called `Test_ttlDestroyHandler_Handle` I am guessing this case is about a pods TTL not having expired. A test names like

`TestTtlDestroyHandlerShouldNotDestroyAPodThatIsNotExpired`
`TestTtlDestroyHandlerShouldDestroyAPodThatIsExpired`

help 😄 

Maybe we should rename `Ttl` to `TTL` according to common Go style 🤔 

Is there any reason why "im-creation-timestamp" is 5min from now and "im-ttl" is 5min?
It caught my attention, but I don't think there is one. If there is none maybe just choose different values so no one has to think about this again 😋